File: storio-sqlite-annotations-processor/src/main/java/com/pushtorefresh/storio/sqlite/annotations/processor/introspection/StorIOSQLiteTypeMeta.java
package com.pushtorefresh.storio.sqlite.annotations.processor.introspection;

import com.pushtorefresh.storio.common.annotations.processor.introspection.StorIOTypeMeta;
import com.pushtorefresh.storio.sqlite.annotations.StorIOSQLiteType;

import org.jetbrains.annotations.NotNull;

public class StorIOSQLiteTypeMeta extends StorIOTypeMeta<StorIOSQLiteType, StorIOSQLiteColumnMeta> {

    public StorIOSQLiteTypeMeta(
            @NotNull String simpleName,
            @NotNull String packageName,
            @NotNull StorIOSQLiteType storIOType,
            boolean needCreator) {
        super(simpleName, packageName, storIOType, needCreator);
    }
}

File: storio-sqlite-annotations-processor/src/main/kotlin/com/pushtorefresh/storio/sqlite/annotations/processor/introspection/StorIOSQLiteTypeMeta.kt
package com.pushtorefresh.storio.sqlite.annotations.processor.introspection

import com.pushtorefresh.storio.common.annotations.processor.introspection.StorIOTypeMeta
import com.pushtorefresh.storio.sqlite.annotations.StorIOSQLiteType

class StorIOSQLiteTypeMeta(
        simpleName: String,
        packageName: String,
        storIOType: StorIOSQLiteType,
        needCreator: Boolean)
    : StorIOTypeMeta<StorIOSQLiteType, StorIOSQLiteColumnMeta>(
        simpleName,
        packageName,
        storIOType,
        needCreator)